libs/packages/sam-formly/src/lib/formly-filters/sds-more-filters.html

```diff
@@ -0,0 +1,42 @@
+<div sds-dialog-title>Advanced Filters</div>
+
```

Comment:
Can you make dialog title configurable

Reply:
as per our discussion with Dave, we don't have to.